### PR TITLE
Location Search

### PR DESCRIPTION
**Pull request by danielpaul**

### CODE REVIEW

**Comment by danielpaul** [comment]
@LevinGeorge @starkshaw The search form didn't work because you had a form inside a form?! 
image

File: db/schema.rb [hunk 1]
```diff
@@ -25,6 +25,15 @@
     t.datetime "updated_at"
   end
 
+  create_table "comments", force: :cascade do |t|
```
Comment (danielpaul) [inline db/schema.rb:65]: Why is this here? :/

**Comment by danielpaul** [comment]
@starkshaw @LevinGeorge Could you pull down and confirm if everything is working and then i'll merge it. Understand the code modifications i've done too.

File: db/schema.rb (hunk 1 above)
Comment (danielpaul) [inline db/schema.rb:65]: Leave this for now...

Comment (starkshaw) [inline db/schema.rb:65]: I think because I merged from master to fix the conflicts.

**Comment by starkshaw** [comment]
One issue is I need to log in to search. I don't think we need that?

**Comment by danielpaul** [comment]
Yeah that shouldn't be the case. Must have missed to modify in the shop controller.

**Comment by starkshaw** [comment]
Can we add some feature of that search field like if user didn't pick address from drop down, then automatically choose first option?

Like if I input "Dubli" and press enter there's nothing in the result.

**Comment by danielpaul** [comment]
#yourNextTask 😜😂

**Comment by starkshaw** [comment]
@danielpaul Don't throw the ball to me again... I just threw it to you... Let me sleep for a while... :zzz:

**Comment by starkshaw** [comment]
The location search is now working.
